routes/api.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var passport = require('passport');
+var router = express.Router();
+
+//router.use(passport.authenticate('bearer', { session: false }));
+
+/* GET api listing. */
+router.get('/getUserInfo', (req, res) => {
+	res.status(200).json({"user": req.user});
```

Comment:
is this a tab or a series of spaces? please be consistent with the use of whitespace